### PR TITLE
Johnhpie

### PR DESCRIPTION
**Pull request by johnhpie**

### CODE REVIEW

**Comment by johnhpie** [comment]
auth test?

**Comment by ericalloyd** [comment]
General comment

File: _slides/16-johnhpie.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Greetings and Salutations, may Robocat blessings be upon you.
```
Comment (ericalloyd) [inline _slides/16-johnhpie.md]: Hey, LGTM 👍🏿

Comment (ericalloyd) [inline _slides/16-johnhpie.md]:
```suggestion
Greetings and Salutations, may Robotocat blessings be upon you.
```

**Review by ericalloyd — CHANGES_REQUESTED** [review]
Suggested change of mascot's name

**Review by ericalloyd — APPROVED** [review]

**Pull request merged**